pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -36,6 +36,9 @@ rec {
     , depsTargetTarget            ? [] #  1 ->  1
     , depsTargetTargetPropagated  ? [] #  1 ->  1
 
+    , checkInputs                 ? []
+    , installCheckInputs          ? []
+
```

Comment:
Needs documentation in nixpkgs manual §3.5

Reply:
Ok. Will do.